### DISCUSSION TITLE
Friends and Guilds

### DISCUSSION THREAD

**Discussion by VV-LX** [ppy/osu#28840] [Ideas] [answered]
A standard Friend/Request System has already been suggested several times, but I’d like to include user created Player Guilds and Mapper Guilds, larger groups of players to serve as in-game hubs for socializing. Think of this as the logical next step for userpage Collaborations. They would also have their own Guild Pages, like the user page for each player.

### Guild-type Challenges, Achievements, and Quests
Exactly what they sound like. Guild-only type quests can be completed, such as dominating the a certain number of spots in a map's leaderboard. This will provide a reward for both the first completing Guild and the server/all who achieve it after. 

### Guild Cosmetics
Sold/obtained through Guild-type Challenges and Events.
To properly affirm the first achiever's unique status, the reward could be based after them or they could receive a slightly different version of it (If it were a medal, maybe the Guild would have a tinted/shiny version, or the ability to display it on their Guild Page)

### Guild Rank & Leaderboard
Completing Guild Quests will add progress in a game-wide ranking/leaderboard for Guilds and the Guild's own internal leaderboards.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/467
